connectors/sources/mssql.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+"""SQLAlchemy source module is responsible to fetch documents from Microsoft SQL."""
+from urllib.parse import quote
+
+from sqlalchemy import create_engine
+
+from connectors.logger import logger
+from connectors.sources.generic_database import GenericBaseDataSource
+
+# Below schemas are system schemas and the tables of the systems schema's will not get indexed
+SYSTEM_SCHEMA = [
+    "INFORMATION_SCHEMA",
+    "db_owner",
+    "db_accessadmin",
+    "db_securityadmin",
+    "db_ddladmin",
+    "db_backupoperator",
+    "db_datareader",
+    "db_datawriter",
+    "db_denydatareader",
+    "db_denydatawriter",
+    "sys",
+]
+
+
+class MSSQLDataSource(GenericBaseDataSource):
+    """Class to fetch documents from Microsoft SQL Server"""
+
+    def __init__(self, connector):
+        """Setup connection to the Microsoft SQL database-server configured by user
+
+        Args:
+            connector (BYOConnector): Object of the BYOConnector class
+        """
+        super().__init__(connector=connector)
+        self.connection_string = f"mssql+pymssql://{self.user}:{quote(self.password)}@{self.host}:{self.port}/{self.database}"
+        self.queries = {
+            "PING": "SELECT 1+1",
+            "ALL_TABLE": "SELECT table_name FROM information_schema.tables WHERE TABLE_SCHEMA = '{schema}'",
```

Comment:
Can we move the queries as global consts to make it easier to read this code?

Reply:
Done